modules/preserve_data.rb

```diff
@@ -0,0 +1,52 @@
+# require './student'
+# require './teacher'
+# require './rental'
```

Comment:
- [ ] Please, kindly remove this old piece of commented code, in order to preserve the cleanliness of the repo.

Reply:
OK Thanks, I would implement the changes